bindings/python/pymongoarrow/types.py

```diff
@@ -88,7 +88,7 @@ def _is_decimal128_str(obj):
     _atypes.is_int64: np.int64,
     _atypes.is_float64: np.float64,
     _atypes.is_timestamp: "datetime64[ms]",
-    _is_objectid: np.object,
+    _is_objectid: object,
```

Comment:
What is the deal with the comment on line 78?:
```
    datetime: lambda _: timestamp("ms"),  # TODO: add tzinfo support
```

Reply:
That was pre-existing, I plan to look at it as part of this PR.

Reply:
It turns out there was nothing to do, I added a note.